package.json

```diff
@@ -4,8 +4,9 @@
   "description": "mini hack to display HDB resale price",
   "main": "index.js",
   "scripts": {
-    "start": "npm run build && npm run start-server",
+    "start": "npm run build && npm run build-index && npm run start-server",
     "start-server": "node dist/server.js",
+    "build-index": "cp public/index.html dist/public/index.html && babel public --out-dir dist/public --presets es2015",
```

Comment:
Heroku build is failing. Logs below. I think need to `mkdir dist/public` before `cp ... dist/public/index.html`

```
2015-12-03T14:51:11.644900+00:00 app[web.1]: src/server.js -> dist/server.js
2015-12-03T14:51:11.691153+00:00 app[web.1]: src/util/hdb.js -> dist/util/hdb.js
2015-12-03T14:51:11.631426+00:00 app[web.1]: src/app.js -> dist/app.js
2015-12-03T14:51:12.235359+00:00 app[web.1]: npm ERR! argv "/app/.heroku/node/bin/node" "/app/.heroku/node/bin/npm" "run" "build-index"
2015-12-03T14:51:12.258556+00:00 app[web.1]: npm ERR! Linux 3.13.0-66-generic
2015-12-03T14:51:12.218076+00:00 app[web.1]: 
2015-12-03T14:51:12.218081+00:00 app[web.1]: > hdb-resale-data@1.0.0 build-index /app
2015-12-03T14:51:12.226270+00:00 app[web.1]: cp: cannot create regular file 'dist/public/index.html': No such file or directory
2015-12-03T14:51:12.235771+00:00 app[web.1]: npm ERR! node v5.0.0
2015-12-03T14:51:12.218083+00:00 app[web.1]: 
2015-12-03T14:51:12.235031+00:00 app[web.1]: npm ERR! Linux 3.13.0-66-generic
2015-12-03T14:51:12.218082+00:00 app[web.1]: > cp public/index.html dist/public/index.html && babel public --out-dir dist/public --presets es2015
2015-12-03T14:51:12.236287+00:00 app[web.1]: npm ERR! npm  v3.3.6
2015-12-03T14:51:12.230842+00:00 app[web.1]: 
2015-12-03T14:51:12.236664+00:00 app[web.1]: npm ERR! Exit status 1
```